client/src/components/RecipeList/components/RecipeCard.jsx

```diff
@@ -1,42 +1,195 @@
+          hoverable
+          cover={(
+            <img
+              className="recipe-image"
+              src={imageURL}
+              alt={title}
+            />
+          )}
+
+          actions={[
+            <Button className="recipe-button" block type="link" size="large" onClick={this.showModal}>
+              <InfoCircleOutlined />
+            </Button>,
+            <Button className="recipe-button" block type="link" size="large">
+              <StarOutlined />
+            </Button>,
+            <Button className="recipe-button" block type="link" size="large" onClick={openRecipe}>
+              <ExportOutlined />
+            </Button>,
+          ]}
+        >
+          <Meta
+            className="recipe-card-meta"
+            title={title}
+            description={site}
+          />
+        </Card>
+        <Modal
+          title={title}
+          visible={showModal}
+          onCancel={this.hideModal}
+          footer={[
+            <Button key="Save">
+              <StarOutlined />
+            </Button>,
+            <Button key="Go" onClick={openRecipe}>
+              <ExportOutlined />
+            </Button>,
+          ]}
+        >
+          <img
+            className="recipe-modal-image"
+            src={imageURL}
+            alt={title}
+          />
+
+          <Divider />
+          <Descriptions title="Recipe Info" layout="vertical" colon={false}>
+            <Descriptions.Item label="Website">{ site }</Descriptions.Item>
+            <Descriptions.Item label="Cooking Time">
+              {totalTime ? `${totalTime} minutes` : (
+                <span>
+                  {' '}
+                  <CloseCircleOutlined />
+                  {' '}
+                  Unavailable
+                  {' '}
+                </span>
+              ) }
+            </Descriptions.Item>
+
+            <Descriptions.Item label="Rating">
+              { rating ? (
+                <span>
+                  {' '}
+                  {parseFloat((rating * 5).toFixed(2))}
+                  /5
+                  {' '}
+                  <StarFilled />
+                  {' '}
+                  from
+                  {' '}
+                  { numberOfRaters }
+                  {' '}
+                  {numberOfRaters > 1 ? 'raters' : 'rater'}
+                  {' '}
+                </span>
+              ) : (
+                <span>
+                  {' '}
+                  <CloseCircleOutlined />
+                  {' '}
+                  Unavailable
+                  {' '}
+                </span>
+              ) }
+            </Descriptions.Item>
+
+            <Descriptions.Item label="Tags" span={3}>
+              {
+                    tags.map((tag, index) => (
+                      <Tag
+                        className="recipe-tag"
+                        visible={index < 5 || viewMore}
+                      >
+                        {' '}
```

Comment:
again, maybe skip the spaces and use CSS instead? you can refer to what I did in the PreferenceTagGroup which has similar CSS to what you're doing here